Resources/views/CRUD/_email_link.html.twig

```diff
@@ -0,0 +1,21 @@
+
+{%- if value is empty -%}
+    &nbsp;
```

Comment:
> Is this really necessary?

@Soullivaneuh I saw something like this in multiple files, for example:
`list_datetime.html.twig`
`list_date.html.twig`
`list_html.html.twig`